.circleci/config.yml

```diff
@@ -0,0 +1,55 @@
+version: 2
+
+env:
+  global:
+  - LC_CTYPE=en_US.UTF-8
+  - LANG=en_US.UTF-8
+
+jobs:
+  swiftlint:
+    docker:
+      - image: dantoml/swiftlint:latest
+    steps:
+      - checkout
+      - run: swiftlint --strict
+
+  test-iOS:
+    macos:
+      xcode: "9.3.0"
+    steps:
+      - checkout
+      - run:
+          name: test iOS
+          command: |
+            set -o pipefail
+            xcodebuild -version
+            xcodebuild -showsdks
+            swift -version
+            sh build.sh test-iOS
+
+  examples:
+    macos:
+      xcode: "9.3.0"
+    steps:
+      - checkout
+      - run:
+          name: examples
+          command: |
+            set -o pipefail
+            xcodebuild -version
+            xcodebuild -showsdks
+            swift -version
+            curl https://cocoapods-specs.circleci.com/fetch-cocoapods-repo-from-s3.sh | bash -s cfk
+            sh build.sh examples
```

Comment:
Looking at the output in the CI, it seems like it's just outputing raw xcodebuild output, maybe it would be possible to pipe it through `xcpretty`?

Reply:
done :)